lib/portal.js

```diff
@@ -9,6 +9,27 @@ const KEYCODES = {
 
 export default class Portal extends React.Component {
 
+  static defaultProps = {
+    onOpen: () => {},
+    onClose: () => {},
+    onUpdate: () => {},
+  };
+
+  static propTypes = {
+    className: React.PropTypes.string,
+    style: React.PropTypes.object,
+    children: React.PropTypes.element.isRequired,
+    target: React.PropTypes.instanceOf(window.HTMLElement),
```

Comment:
What do you think about receive a string as `targetSelector` and query this element by `document.querySelector` on `componentDidMount`?